activemodel/lib/active_model/type/helpers/accepts_multiparameter_time.rb

```diff
@@ -3,42 +3,46 @@
 module ActiveModel
   module Type
     module Helpers # :nodoc: all
-      class AcceptsMultiparameterTime < Module
-        def initialize(defaults: {})
-          define_method(:serialize) do |value|
-            super(cast(value))
-          end
+      module AcceptsMultiparameterTime
+        extend ActiveSupport::Concern
 
-          define_method(:cast) do |value|
-            if value.is_a?(Hash)
-              value_from_multiparameter_assignment(value)
-            else
-              super(value)
-            end
-          end
+        included do
+          class_attribute :defaults, instance_writer: false, default: {}
```

Comment:
Would it be good for this to have a more specific name now, like `multiparameter_defaults`?

Reply:
I've changed the `defaults` class attribute to the private `_defaults` instance method. Is it fine enough?

https://github.com/rails/rails/blob/3fcc4fe66fce913ba371192c0a92fcee427775b5/activemodel/lib/active_model/type/helpers/accepts_multiparameter_time.rb#L7-L11

Reply:
https://github.com/rails/rails/pull/39645/commits/b570f5343f6690aea503af7d6f28b9d56c113ed0 looks even better!  :+1: :+1: